tests/foreman/api/test_contentmanagement.py

```diff
@@ -1108,3 +1109,104 @@ def test_positive_sync_container_repo_end_to_end(
                 f'{con_client} logout {capsule_configured.hostname}'
             )
             assert result.status == 0
+
+    @pytest.mark.tier4
+    @pytest.mark.skip_if_not_set('capsule')
+    def test_positive_sync_collection_repo(
+        self, default_sat, module_org, module_product, capsule_configured, rhel7_contenthost
+    ):
+        """Sync ansible-collection repo to the capsule, consume it on a host.
+
+        :id: cfca7b28-fb70-429a-9aa2-db0d8aaf99bc
+
+        :steps:
+            1. Create an ansible-collection type repository.
+            2. Sync the repository to the capsule through the Library LCE.
+               (CVs are unsupported yet)
+            3. Check the Capsule is synced without errors.
+            4. Try to install the collections from the Capsule on a contenthost.
+
+        :expectedresults:
+            1. The ansible-collection repo is successfully synced to the capsule.
+            2. The collection is successfully installed on the contenthost.
+
+        :CaseLevel: Integration
+        """
+        requirements = '''
+        ---
+        collections:
+        - name: theforeman.foreman
+          version: "2.1.0"
+        - name: theforeman.operations
+          version: "0.1.0"
+        '''
+        repo = entities.Repository(
+            content_type='ansible_collection',
+            ansible_collection_requirements=requirements,
+            product=module_product,
+            url=ANSIBLE_GALAXY,
+        ).create()
+
+        # Find "Library" lifecycle env for specific organization
+        lce = entities.LifecycleEnvironment(organization=module_org).search(
```

Comment:
If organization=module_org, can you not use the fixture `def module_lce_library(module_org)` in `pytest_fixtures/component/lce.py`?

Reply:
I think I could, thank you for pointing this out. :+1: